blog/2021-09-15-soc2-things-i-learned-part-1.md

```diff
@@ -0,0 +1,142 @@
+
+In this respect, credit is due where credit is earned.
+Strike Graph's "risk-based" approach did help us build a substantial foundation to work from.
+Their team provided templates for policies, and also put us in touch with an auditing firm.
+At the time, this was super helpful, and the effort was successful.
+
+In February of 2021 we received a clean SOC 2 Type 1 Security report issued for January 31, 2021.
+However, by the end of our audit, the smooth experience had eroded significantly and we discovered some truths about what we needed to be successful in the eyes of the SOC 2 framework moving forward.
+
+As we looked towards the Type 2, we evaluated the bumps we encountered towards the end of our Type 1 audit.
+While we had all increased our understanding of the SOC 2 process, we needed a better understanding of how our internal repeatable processes should be captured as control statements and how those control statements mapped to criteria.
+
+### Martin Cozzi
+
+[Martin Cozzi](https://www.marana.io/) is a consultant who specializes in the SOC 2 compliance process, and we were extremely fortunate to have retained Martin's services going into our Type 2 period.
+
+Its probably not surprising in retrospect, but the biggest thing we learned from Martin, is that the internal culture of the organization, and its fundamental understanding and commitment to the SOC 2 framework, is what ultimately makes an audit feel like a smooth routine exercise, versus a slogging and demoralizing haze.
+Martin helped us understand that we needed to embrace SOC2, not outsource it.
+For an organization, we learned, SOC 2 compliance essentially affects our day to day way of life.
+
+The other thing that Martin did, which is truly an art form, and where third party services like Strike Graph struggle, is writing custom Control Statements.
+For a Type 1, Control Statements can be broad, less than accurate, or sometimes irrelevant, and you can still get by.
+For ongoing Type 2 compliance, Control Statements should be super accurate, concise, and specific to the organization's systems.
+
+I asked Martin once, "is there a course that I can take to learn to do this?".
+"Unfortunately, no, at least not yet", he said.
+
+Martin explained that he learned how to write control statements from years of experience working in the industry and working through dozens of audits.
+To do it effectively, requires an in-depth understanding of the organization, the SOC 2 criteria, and the auditing partner's expectations.
+
+With Martin's guidance, we were able to dial-in our internal processes, establish a compliance-responsibility hierarchy, and curate a rock solid set of control statements.
+
+Not only did we manage to execute a Type 2 audit with very little friction, we managed to do it, not just for Security, but Availability and Confidentiality as well.
+
+---
+
+Stay tuned for the next post in this series.
```

Comment:
1. ask people to subscribe to mailing list, don't just "stay tuned"
2. what is the next post? please tease.